src/app/core/entity/entity-mapper.service.ts

```diff
@@ -102,22 +102,27 @@ export class EntityMapperService {
    *
    * <em>Important:</em> Loading via the constructor is always preferred compared to loading via string. The latter
    * doesn't allow strict type-checking and errors can only be discovered later
-   * The first update that one will receive - immediately after subscribing - is <code>null</code>.
-   * The <code>update</code>-function takes this into account.
    * @param entityType the type of the entity or the registered name of that class.
    */
   public receiveUpdates<T extends Entity>(
```

Comment:
This could be extended so you can receive updates for **all** entity types or a list of entity types.